adk-cymbal-retail-agent/proxies/cymbal-customers-v1/apiproxy/proxies/default.xml

```diff
@@ -38,7 +38,7 @@
         </Step>
       </Request>
       <Response/>
-      <Condition>(proxy.pathsuffix MatchesPath "/") and (request.verb = "GET")</Condition>
+      <Condition>((request.path MatchesPath "/v1/samples/adk-cymbal-retail/customers" or request.path MatchesPath "/v1/samples/adk-cymbal-retail/customers/")) and (request.verb = "GET")</Condition>
```

Comment:
Although this works, there's redundant (), I like this style.

<Condition>((request.path MatchesPath "/v1/samples/adk-cymbal-retail/customers") or (request.path MatchesPath "/v1/samples/adk-cymbal-retail/customers/")) and (request.verb = "GET")</Condition>